packages/docusaurus-plugin-content-pages/index.js

```diff
@@ -31,6 +29,10 @@ class DocusaurusPluginContentPages {
     return 'docusaurus-plugin-content-pages';
   }
 
+  getPathsToWatch() {
+    return [this.contentPath];
```

Comment:
WDYT of optimizing this to be specific to the extensions included ?

This is because `fs.watcher('someDirectory')` and  `fs.watcher('someDirectory/**/*.js') `can be difference in performance

Reply:
I see. Let's do that in future. I've added it to the HackMD.